netbox/extras/management/commands/dev.py

```diff
@@ -0,0 +1,22 @@
+import typing as t
+import threading
+from django.core.management import call_command
+from django.contrib.staticfiles.management.commands import runserver
+
+
+class Command(runserver.Command):
+    """Run the NetBox development server.
+
+    Builds UI source files, runs `collectstatic`, starts the Django development web server, and
+    watches UI source files for changes.
+    """
+
+    def handle(self, *args: t.Any, **options: t.Any) -> t.Optional[str]:
+        # Run the UI development server in a separate thread so file watching doesn't block the
+        # Django development server running in the main thread.
+        ui_dev = threading.Thread(name="uidev", target=lambda: call_command("uidev"))
+        ui_dev.start()
+        # Run `collectstatic`.
+        call_command("collectstatic", interactive=False)
```

Comment:
Per the previous discussion, `collectstatic` should not be needed, as the `runserver` daemon [automatically serves static files](https://docs.djangoproject.com/en/4.0/ref/contrib/staticfiles/#staticfiles-runserver) from their source locations. (If this does not appear to be working locally, further investigation is needed to determine the root cause.)